### PR TITLE
ignore null tablespace

### PR DESCRIPTION
**Pull request by pivnicek**
for https://github.com/keboola/db-extractor-oracle/issues/73

### CODE REVIEW

**Comment by pivnicek** [comment]
FYI, expected results diff is 
image

File: src/main/java/com/keboola/tableexporter/MetaFetcher.java [hunk 1]
```diff
@@ -79,7 +79,9 @@ public TreeMap fetchTableListing() throws UserException {
     public LinkedHashMap getTableData(ResultSet rs) throws SQLException {
         LinkedHashMap tableData = new LinkedHashMap();
         tableData.put("name", rs.getString("TABLE_NAME"));
-        tableData.put("tablespaceName", rs.getString("TABLESPACE_NAME"));
```
Comment (odinuv) [inline src/main/java/com/keboola/tableexporter/MetaFetcher.java]: woudn't it be better to cast it to empty string?

Comment (pivnicek) [reply src/main/java/com/keboola/tableexporter/MetaFetcher.java]: why?  null means missing, right?

Comment (odinuv) [reply src/main/java/com/keboola/tableexporter/MetaFetcher.java]: right, but then you can't pick a table with no tablespacename

or we would have to need to search a table by non-presence of metadata?

Comment (pivnicek) [reply src/main/java/com/keboola/tableexporter/MetaFetcher.java]: yes, I think a simple `hasKey(key)` can work with that.

**Pull request merged**